.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,49 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**I confirm:**
+<!-- To mark a task as complete, use [x] -->
+- [ ] I haven't found an existing/similar bug report for my bug.
+- [ ] this bug report is about a single actionable bug.
+- [ ] if this issue has been spotted on the git-scm.com site, I've cross-checked that my issue is still present in the current pdf version.
+- [ ] this bug is about the second version of the Git Pro book in English. A bug for a translation should be filed with the translation project. You can find a table with those projects here: [progit2/TRANSLATING.md](https://github.com/progit/progit2/blob/master/TRANSLATING.md)
+- [ ] this bug is not about the git-scm.com site, if so please file a bug here: [git-scm.com/issues/new](https://github.com/git/git-scm.com/issues/new)
```

Comment:
At the same time, the only accepted bugs are the pdfs or git-scm pages. Other compilations of the book are not supported.

Reply:
I've made a new commit, it includes the line:
```
- [ ] this bug is **not** about .epub or .mobi files.
```